pkg/client/unversioned/helper.go

```diff
@@ -414,6 +414,31 @@ func RESTClientFor(config *Config) (*RESTClient, error) {
 	return client, nil
 }
 
+// UnversionedRESTClientFor is the same as RESTClientFor, except that it allows
+// the config.Version to be empty.
+func UnversionedRESTClientFor(config *Config) (*RESTClient, error) {
```

Comment:
@deads2k 
I forked another constructor that doesn't do the empty version check.